airflow/providers/cncf/kubernetes/triggers/pod.py

```diff
@@ -92,6 +93,7 @@ def __init__(
         in_cluster: bool | None = None,
         get_logs: bool = True,
         startup_timeout: int = 120,
+        startup_check_interval_seconds: int = 1,
```

Comment:
Can we increase this (5/10 seconds) to avoid to reduce the number of calls to K8 API server?

Reply:
In the operator, the default is set to 1  so I kept the same here. But if you want I can change it here as well in the operator

Reply:
updated to 5 seconds